third_party/move/move-compiler/src/inlining/translate.rs

```diff
@@ -1094,17 +1100,24 @@ fn get_args_from_exp(args: &Exp) -> Vec<Exp> {
     }
 }
 
-fn get_params_from_decls(decls: &LValueList) -> Vec<Symbol> {
+fn get_params_from_decls(inliner: &mut Inliner, decls: &LValueList) -> Vec<Symbol> {
     decls
         .value
         .iter()
         .flat_map(|lv| match &lv.value {
-            LValue_::Var(v, _) => vec![v.0.value],
-            LValue_::Ignore => vec![],
+            LValue_::Var(v, _) => vec![Some(v.0.value)],
+            LValue_::Ignore => vec![None], // placeholder for "_"
```

Comment:
Is this a placeholder for only `_` or any var name that begins with a `_`?

Reply:
`_` is treated specially in the parser.  Symbols starting with `_` are only treated specially when checking whether they are unused.  Only `_` becomes `Ignore`.